agent_api_rest/postman/ssi-agent.postman_collection.json

```diff
@@ -249,7 +249,125 @@
 					],
 					"request": {
 						"method": "POST",
-						"header": [],
+						"header": [
+							{
+								"key": "Access-Control-Allow-Origin",
```

Comment:
Why do we need the CORS header?

Reply:
That shouldn't be there, I'm removing it :+1: